assets/prototype/domain/eventEditor/datesList/dateCard/DateCard.js

```diff
@@ -55,20 +55,8 @@ const DateCard = ({ eventId, id, tickets }) => {
 		<DateTimeProvider id={id}>
 			<Card elevation={Elevation.ONE} style={cardStyle}>
 				<EditDate position='top' relatedTickets={relatedTicketIds} tickets={tickets} />
-				<div>
-					<div style={idStyle}>
-						{date.datetimeId} {':'} {date.id}
-					</div>
-					<H4>
-						<EditableText
-							placeholder='Edit title...'
-							defaultValue={date.name}
-							onCancel={(value) => console.log(value)}
-							onConfirm={(name) => onFieldUpdate({ name })}
-							minWidth={'320px'}
-							selectAllOnFocus
-						/>
-					</H4>
```

Comment:
This is a duplicate.
Check code below :point_down: 
![image](https://user-images.githubusercontent.com/1477453/70074385-41085700-1603-11ea-9de6-064e4e75fe9e.png)

Reply:
😆